app/src/main/java/io/homeassistant/companion/android/notifications/MessagingManager.kt

```diff
@@ -1287,6 +1287,18 @@ class MessagingManager @Inject constructor(
             return@withContext FileProvider.getUriForFile(context, "${context.packageName}.provider", file)
         }
 
+    private suspend fun isGif(url: URL): Boolean {
+        try {
+            val connection = url.openConnection()
+            connection.setRequestProperty("User-Agent", "Mozilla/5.0")
```

Comment:
Any reason for setting the user agent, and if so why not use the [app's user agent] (https://github.com/home-assistant/android/blob/master/common/src/main/java/io/homeassistant/companion/android/common/data/HomeAssistantApis.kt#L27)?

Reply:
Not really. Just thought, that this is a good general one. But you are probably right. Il will commit it!

Reply:
Do you have an idea how to import the BuildConfig.VERSION_NAME? Seems not imported.

Reply:
Better to import the user agent variable I linked than to copy the contents. Add:
```kotlin
import io.homeassistant.companion.android.common.data.HomeAssistantApis
```
and use:
```kotlin
connection.setRequestProperty("User-Agent", HomeAssistantApis.USER_AGENT_STRING)
```

Reply:
done